controllers/cloud_config_sync_controller.go

```diff
@@ -0,0 +1,196 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog"
+	"reflect"
```

Comment:
Can we group the imports so that core imports are in their own group please? Shouldn't we have a lint rule for goimports somewhere?